airflow/providers/amazon/aws/auth_manager/avp/facade.py

```diff
@@ -205,3 +245,25 @@ def _build_context(context: dict | None) -> dict | None:
         return {
             "contextMap": context,
         }
+
+    def _build_is_authorized_request_payload(self, request: IsAuthorizedRequest, user: AwsAuthManagerUser):
+        """
+        Build payload of individual authorization request send through ``batch_is_authorized`` API.
```

Comment:
This is what I think this is trying to say, is that right?

```suggestion
        Build a payload of an individual authorization request that could be sent through the ``batch_is_authorized`` API.
```

Reply:
Correct, thanks :)